### PR TITLE
CloudCare Debugger UX Revamp

### PR DESCRIPTION
**Pull request by benrudolph**
@czue @wpride this revamps the cc debugger to be more user friendly (imo). ux peeps: @biyeun @orangejenny @dimagi/product 

screenshots:
screen shot 2015-11-26 at 8 40 30 pm
screen shot 2015-11-26 at 8 40 11 pm
screen shot 2015-11-26 at 8 40 24 pm
screen shot 2015-11-26 at 8 40 21 pm
screen shot 2015-11-26 at 8 40 17 pm

### CODE REVIEW

**Comment by benrudolph** [comment]
https://github.com/dimagi/touchforms/pull/185

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js [hunk 1]
```diff
@@ -704,6 +707,12 @@ cloudCare.AppView = Backbone.View.extend({
                             readOnly: true,
                             lineNumbers: true,
                         });
+$('#debugger-xml-instance-tab')
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: Missing semicolon.

**Comment by benrudolph** [comment]
the latest in a series of PRs coming from procrastination when working on my training presentation

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js [hunk 2]
```diff
@@ -705,6 +708,11 @@ cloudCare.AppView = Backbone.View.extend({
                             lineNumbers: true,
                         });
 
+                        $instanceTab.off();
```
Comment (benrudolph) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: this is ugly, but in order to not put this here, we need to do a pretty invasive refactor the debugger so i punted (debugger should belong in `fullform-ui.js`)

Comment (wpride) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: I'm thankful for Ben

**Comment by benrudolph** [comment]
fyi, both PRs can be read :blowfish: 

watch out world, cloudcare is about to take you by storm! :zap: :zap:

**Comment by czue** [comment]
this is awesome!

**Comment by czue** [comment]
:+1: from me, but didn't review code super closely. will let @wpride look before merging.

**Comment by snopoke** [comment]
sweet like :candy:

**Comment by biyeun** [comment]
hot potato :sweet_potato:

**Comment by benrudolph** [comment]
i don't think this a huge change (everything is behind a feature flag). would be cool to get this in, i think will is traveling this week. would be happy to followup and any suggestions he has

**Pull request merged**